src/adapter/src/catalog/error.rs

```diff
@@ -134,6 +135,12 @@ impl From<mz_stash::StashError> for Error {
     }
 }
 
+impl From<TryFromProtoError> for Error {
+    fn from(e: TryFromProtoError) -> Error {
```

Comment:
This seems unused, might be interesting to test the error case?

Reply:
Let me come back to this, I'm pretty sure it's used in the implementation of `TableTransaction` which is part of `mz_catalog::Transaction`, so I would expect it to be tested 🤔

Reply:
Easy way to verify is adding a panic in here and running the test you expect to cover it.